src/SimpleFileUpload.tsx

```diff
@@ -1,28 +1,32 @@
 import * as React from 'react';
 import { FieldProps, getIn } from 'formik';
 import FormControl from '@material-ui/core/FormControl';
-import InputLabel from '@material-ui/core/InputLabel';
-import Input from '@material-ui/core/Input';
+import InputLabel, { InputLabelProps } from '@material-ui/core/InputLabel';
+import Input, { InputProps } from '@material-ui/core/Input';
 import FormHelperText from '@material-ui/core/FormHelperText';
 
 export interface SimpleFileUploadProps extends FieldProps {
   label: string;
   disabled?: boolean;
+  inputProps: InputProps;
+  inputLabelProps: InputLabelProps;
```

Comment:
both should be optional. E.g. `inputLabelProps?: InputLabelProps;`

Reply:
Change made